targets/nuttx-stm32f4/nuttx/config_mpu

```diff
@@ -0,0 +1,1673 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
Well, my key point was consistency. If dot-files are disliked, then the existing config files (their names) should be adapted. (I do know `ls`. I even know `ls -a`.)